feature_engine/creation/mathematical_combination.py

```diff
@@ -251,6 +259,9 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         for new_variable_name, operation in self.combination_dict_.items():
             X[new_variable_name] = X[self.variables_to_combine].agg(operation, axis=1)
 
+        if self.drop_original:
+            X.drop(columns=self.variables_, inplace=True)
```

Comment:
here we need to drop `self.variables_to_combine`.

Reply:
> 

Can you provide an example of how to write a test? Thank you

Reply:
In this [code block](https://github.com/feature-engine/feature_engine/blob/main/tests/test_creation/test_cyclical_transformer.py#L75), we extend the first test of the cyclical transformer to test the drop_original attribute.

In short, I would suggest you pick one of the tests that already exist for each of the transformers, and replicate it, changing the expected output when drop_original=True